erpnext/erpnext_integrations/taxjar_integration.py

```diff
@@ -77,11 +80,11 @@ def create_transaction(doc, method):
 			update_integration_request(integration_request, endpoint="client.create_order")
 			response = client.create_order(tax_dict)
 	except taxjar.exceptions.TaxJarResponseError as err:
-		update_integration_request(integration_request, status="Failed", error=str(err))
-		error_msg = """There seems to be an error in Address<br><br>{0}""".format(sanitize_error_response(err))
-		frappe.throw(_(error_msg), AddressError, _("Address Error"))
-	except Exception as ex:
-		print(traceback.format_exc(ex))
+		update_integration_request(integration_request, status="Failed", error=sanitize_error_response(err))
+		error_msg = """There seems to be an error <br><br>{0}""".format(sanitize_error_response(err))
```

Comment:
use _() to translate the string and `<br><br>`  will cause problems during translation. will recommend to add tags on a translated string.